<html xmlns="http://www.w3.org/1999/xhtml">
<head xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml"><title>Genomics and Proteomics</title><meta name="created-time" content="2012/08/23 14:04:51 -0500"/><meta name="revised-time" content="2013/04/11 13:25:29.093 GMT-5"/><meta name="author" content="OpenStaxCollege"/><meta name="licensor" content="OSCRiceUniversity"/><meta name="license" content="http://creativecommons.org/licenses/by/3.0/"/><meta name="keywords" content="biomarker, biotechnology, false negative, genomics, metabolome, metabolomics, protein signature, proteomics, systems biology"/><meta name="subject" content="Science and Technology"/></head>

<body xmlns="http://www.w3.org/1999/xhtml" xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml" xmlns:qml="http://cnx.rice.edu/qml/1.0" xmlns:mod="http://cnx.rice.edu/#moduleIds" xmlns:bib="http://bibtexml.sf.net/" xmlns:data="http://dev.w3.org/html5/spec/#custom"><div data-type="document-title">Genomics and Proteomics</div><div data-type="abstract">By the end of this section, you will be able to:
<ul>
<li>Explain systems biology</li>
<li>Describe a proteome</li>
<li>Define protein signature</li>
</ul></div>
<p id="fs-id1047239">Proteins are the final products of genes, which help perform the function encoded by the gene. Proteins are composed of amino acids and play important roles in the cell. All enzymes (except ribozymes) are proteins that act as catalysts to affect the rate of reactions. Proteins are also regulatory molecules, and some are hormones. Transport proteins, such as hemoglobin, help transport oxygen to various organs. Antibodies that defend against foreign particles are also proteins. In the diseased state, protein function can be impaired because of changes at the genetic level or because of direct impact on a specific protein.</p>
<p id="fs-id2140999">A <span data-type="term">proteome</span> is the entire set of proteins produced by a cell type. Proteomes can be studied using the knowledge of genomes because genes code for mRNAs, and the mRNAs encode proteins. Although mRNA analysis is a step in the right direction, not all mRNAs are translated into proteins. The study of the function of proteomes is called <span data-type="term">proteomics</span>. Proteomics complements genomics and is useful when scientists want to test their hypotheses that were based on genes. Even though all cells of a multicellular organism have the same set of genes, the set of proteins produced in different tissues is different and dependent on gene expression. Thus, the genome is constant, but the proteome varies and is dynamic within an organism. In addition, RNAs can be alternately spliced (cut and pasted to create novel combinations and novel proteins) and many proteins are modified after translation by processes such as proteolytic cleavage, phosphorylation, glycosylation, and ubiquitination. There are also protein-protein interactions, which complicate the study of proteomes. Although the genome provides a blueprint, the final architecture depends on several factors that can change the progression of events that generate the proteome.</p>
<p id="fs-id1436311">Metabolomics is related to genomics and proteomics. <span data-type="term">Metabolomics</span> involves the study of small molecule metabolites found in an organism. The <span data-type="term">metabolome</span> is the complete set of metabolites that are related to the genetic makeup of an organism. Metabolomics offers an opportunity to compare genetic makeup and physical characteristics, as well as genetic makeup and environmental factors. The goal of metabolome research is to identify, quantify, and catalogue all of the metabolites that are found in the tissues and fluids of living organisms.</p>

<section data-depth="1" id="fs-id3070686"><h1 data-type="title">Basic Techniques in Protein Analysis</h1>
<p id="fs-id1699749">The ultimate goal of proteomics is to identify or compare the proteins expressed from a given genome under specific conditions, study the interactions between the proteins, and use the information to predict cell behavior or develop drug targets. Just as the genome is analyzed using the basic technique of DNA sequencing, proteomics requires techniques for protein analysis. The basic technique for protein analysis, analogous to DNA sequencing, is mass spectrometry. Mass spectrometry is used to identify and determine the characteristics of a molecule. Advances in spectrometry have allowed researchers to analyze very small samples of protein. X-ray crystallography, for example, enables scientists to determine the three-dimensional structure of a protein crystal at atomic resolution. Another protein imaging technique, nuclear magnetic resonance (NMR), uses the magnetic properties of atoms to determine the three-dimensional structure of proteins in aqueous solution. Protein microarrays have also been used to study interactions between proteins. Large-scale adaptations of the basic two-hybrid screen (<a href="#fig-chF17_05_01" class="autogenerated-content">[link]</a>) have provided the basis for protein microarrays. Computer software is used to analyze the vast amount of data generated for proteomic analysis.</p><p id="fs-id3096087">Genomic- and proteomic-scale analyses are part of systems biology. <span data-type="term">Systems biology</span> is the study of whole biological systems (genomes and proteomes) based on interactions within the system. The European Bioinformatics Institute and the Human Proteome Organization (HUPO) are developing and establishing effective tools to sort through the enormous pile of systems biology data. Because proteins are the direct products of genes and reflect activity at the genomic level, it is natural to use proteomes to compare the protein profiles of different cells to identify proteins and genes involved in disease processes. Most pharmaceutical drug trials target proteins. Information obtained from proteomics is being used to identify novel drugs and understand their mechanisms of action.</p>
<figure id="fig-chF17_05_01"><figcaption>Two-hybrid screening is used to determine whether two proteins interact. In this method, a transcription factor is split into a DNA-binding domain (BD) and an activator domain (AD). The binding domain is able to bind the promoter in the absence of the activator domain, but it does not turn on transcription. A protein called the bait is attached to the BD, and a protein called the prey is attached to the AD. Transcription occurs only if the prey &#8220;catches&#8221; the bait.</figcaption><span data-type="media" id="fs-id2569068" data-alt="In two-hybrid screening, the binding domain of a transcription factor is separated from the activator domain. A bait protein is attached to the DNA-binding domain of a transcription factor, and a prey protein is attached to the activator domain. If the prey catches the bait (in other words, binds to it), transcription of a reporter gene occurs. If the prey does not catch the bait, no transcription occurs. Scientists use this transcriptional activation to determine if interaction between the bait and prey has occurred.">
<img src="/resources/9bffbde2c927187f0841adea182fca8ed4ce7bf3/Figure_17_05_01.jpg" data-media-type="image/jpg" alt="In two-hybrid screening, the binding domain of a transcription factor is separated from the activator domain. A bait protein is attached to the DNA-binding domain of a transcription factor, and a prey protein is attached to the activator domain. If the prey catches the bait (in other words, binds to it), transcription of a reporter gene occurs. If the prey does not catch the bait, no transcription occurs. Scientists use this transcriptional activation to determine if interaction between the bait and prey has occurred." width="350"/>
</span>
</figure><p id="fs-id2024686">The challenge of techniques used for proteomic analyses is the difficulty in detecting small quantities of proteins. Although mass spectrometry is good for detecting small amounts of proteins, variations in protein expression in diseased states can be difficult to discern. Proteins are naturally unstable molecules, which makes proteomic analysis much more difficult than genomic analysis.</p>
</section>

<section data-depth="1" id="fs-id2874195"><h1 data-type="title">Cancer Proteomics</h1>
<p id="fs-id1443636">Genomes and proteomes of patients suffering from specific diseases are being studied to understand the genetic basis of the disease. The most prominent disease being studied with proteomic approaches is cancer. Proteomic approaches are being used to improve screening and early detection of cancer; this is achieved by identifying proteins whose expression is affected by the disease process. An individual protein is called a <span data-type="term">biomarker</span>, whereas a set of proteins with altered expression levels is called a <span data-type="term">protein signature</span>. For a biomarker or protein signature to be useful as a candidate for early screening and detection of a cancer, it must be secreted in body fluids, such as sweat, blood, or urine, such that large-scale screenings can be performed in a non-invasive fashion. The current problem with using biomarkers for the early detection of cancer is the high rate of false-negative results. A <span data-type="term">false negative</span> is an incorrect test result that should have been positive. In other words, many cases of cancer go undetected, which makes biomarkers unreliable. Some examples of protein biomarkers used in cancer detection are CA-125 for ovarian cancer and PSA for prostate cancer. Protein signatures may be more reliable than biomarkers to detect cancer cells. Proteomics is also being used to develop individualized treatment plans, which involves the prediction of whether or not an individual will respond to specific drugs and the side effects that the individual may experience. Proteomics is also being used to predict the possibility of disease recurrence.</p>
<p id="fs-id2072285">The National Cancer Institute has developed programs to improve the detection and treatment of cancer. The Clinical Proteomic Technologies for Cancer and the Early Detection Research Network are efforts to identify protein signatures specific to different types of cancers. The Biomedical Proteomics Program is designed to identify protein signatures and design effective therapies for cancer patients.</p>
</section>

<section data-depth="1" id="fs-id1373933" class="summary"><h1 data-type="title">Section Summary</h1><p id="fs-id1463775">Proteomics is the study of the entire set of proteins expressed by a given type of cell under certain environmental conditions. In a multicellular organism, different cell types will have different proteomes, and these will vary with changes in the environment. Unlike a genome, a proteome is dynamic and in constant flux, which makes it both more complicated and more useful than the knowledge of genomes alone.</p>
<p id="fs-id3089573">Proteomics approaches rely on protein analysis; these techniques are constantly being upgraded. Proteomics has been used to study different types of cancer. Different biomarkers and protein signatures are being used to analyze each type of cancer. The future goal is to have a personalized treatment plan for each individual.</p>
</section>

<section data-depth="1" id="fs-id2344950" class="multiple-choice"><h1 data-type="title">Review Questions</h1><div data-type="exercise" class="exercise" id="fs-id1956957">
<div data-type="problem" class="problem" id="fs-id1798390"><p id="fs-id2334765">What is a biomarker?</p>
<ol id="fs-id2010215" data-number-style="lower-alpha">
<li>the color coding of different genes</li>
<li>a protein that is uniquely produced in a diseased state</li>
<li>a molecule in the genome or proteome</li>
<li>a marker that is genetically inherited</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id2694573" data-label=""> <p id="fs-id2595588">B</p> </div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1236114">
<div data-type="problem" class="problem" id="fs-id2137700"><p id="fs-id2338028">A protein signature is:</p>
<ol id="fs-id2335177" data-number-style="lower-alpha"><li>the path followed by a protein after it is synthesized in the nucleus</li>
<li>the path followed by a protein in the cytoplasm</li>
<li>a protein expressed on the cell surface</li>
<li>a unique set of proteins present in a diseased state</li>
</ol></div>
<div data-type="solution" class="solution" id="fs-id2890478" data-label=""> <p id="fs-id1201883">D</p> </div>
</div>
</section>
<section data-depth="1" id="fs-id1812902" class="free-response"><h1 data-type="title">Free Response</h1>
<div data-type="exercise" class="exercise" id="fs-id2571119">
<div data-type="problem" class="problem" id="fs-id2322326"><p id="fs-id1867113">How has proteomics been used in cancer detection and treatment?</p></div>
<div data-type="solution" class="solution" id="fs-id1802723" data-label="">
<p id="fs-id2168046">Proteomics has provided a way to detect biomarkers and protein signatures, which have been used to screen for the early detection of cancer.</p></div>
</div>
<div data-type="exercise" class="exercise" id="fs-id2491698"><div data-type="problem" class="problem" id="fs-id2626112"><p id="fs-id2570594">What is personalized medicine?</p></div>
<div data-type="solution" class="solution" id="fs-id1293715" data-label=""><p id="fs-id2569349">Personalized medicine is the use of an individual's genomic sequence to predict the risk for specific diseases. When a disease does occur, it can be used to develop a personalized treatment plan.</p></div>
</div></section>
<div data-type="glossary"><h2 data-type="glossary-title">Glossary</h2>
<dl class="definition" id="fs-id2100648"><dt>biomarker</dt> <dd id="fs-id1424024">individual protein that is uniquely produced in a diseased state</dd></dl>
<dl class="definition" id="fs-id2925815"><dt>false negative</dt> <dd id="fs-id2278509">incorrect test result that should have been positive</dd></dl>
<dl class="definition" id="fs-id2596155"><dt>metabolome</dt> <dd id="fs-id2694643">complete set of  metabolites which are related to the genetic makeup of an organism</dd></dl>
<dl class="definition" id="fs-id1555243"><dt>metabolomics</dt> <dd id="fs-id2098378">study of small molecule metabolites found in an organism</dd></dl>
<dl class="definition" id="fs-id2570734"><dt>protein signature</dt> <dd id="fs-id1786911">set of uniquely expressed proteins in the diseased state</dd></dl>
<dl class="definition" id="fs-id1988758"><dt>proteome</dt> <dd id="fs-id2279649">entire set of proteins produced by a cell type</dd></dl>
<dl class="definition" id="fs-id2155997"><dt>proteomics</dt> <dd id="fs-id2867696">study of the function of proteomes</dd></dl>
<dl class="definition" id="fs-id2937179"><dt>systems biology</dt> <dd id="fs-id1968258">study of whole biological systems (genomes and proteomes) based on interactions within the system</dd></dl>
</div></body>

</html>