<html xmlns="http://www.w3.org/1999/xhtml">
<head xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml"><title>DNA Replication in Prokaryotes</title><meta name="created-time" content="2012/08/23 08:43:08 -0500"/><meta name="revised-time" content="2016/02/25 15:59:52.836 US/Central"/><meta name="author" content="OpenStaxCollege"/><meta name="licensor" content="OSCRiceUniversity"/><meta name="license" content="http://creativecommons.org/licenses/by/4.0/"/><meta name="keywords" content="deoxyribonucleic acid, DNA, DNA function, DNA replication, DNA structure, enzyme function, helicase, lagging strand, leading strand, ligase, Okazaki fragment, primase, primer, prokaryotic DNA, replication fork, single-strand binding protein, sliding clamp, topoisomerase"/><meta name="subject" content="Science and Technology"/></head>

<body xmlns="http://www.w3.org/1999/xhtml" xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml" xmlns:qml="http://cnx.rice.edu/qml/1.0" xmlns:mod="http://cnx.rice.edu/#moduleIds" xmlns:bib="http://bibtexml.sf.net/" xmlns:data="http://www.w3.org/TR/html5/dom.html#custom-data-attribute"><div data-type="document-title">DNA Replication in Prokaryotes</div><div data-type="abstract">By the end of this section, you will be able to:
<ul>
<li>Explain the process of DNA replication in prokaryotes</li>
<li>Discuss the role of different enzymes and proteins in supporting this process</li>
</ul></div>
<p id="fs-id2571092">DNA replication has been extremely well studied in prokaryotes primarily because of the small size of the genome and the mutants that are available. <em data-effect="italics">E. coli </em>has 4.6 million base pairs in a single circular chromosome and all of it gets replicated in approximately 42 minutes, starting from a single origin of replication and proceeding around the circle in both directions. This means that approximately 1000 nucleotides are added per second. The process is quite rapid and occurs without many mistakes.</p>
<p id="fs-id1806393">DNA replication employs a large number of proteins and enzymes, each of which plays a critical role during the process. One of the key players is the enzyme DNA polymerase, also known as DNA pol, which adds nucleotides one by one to the growing DNA chain that are complementary to the template strand. The addition of nucleotides requires energy; this energy is obtained from the nucleotides that have three phosphates attached to them, similar to ATP which has three phosphate groups attached. When the bond between the phosphates is broken, the energy released is used to form the phosphodiester bond between the incoming nucleotide and the growing chain. In prokaryotes, three main types of polymerases are known: DNA pol I, DNA pol II, and DNA pol III. It is now known that DNA pol III is the enzyme required for DNA synthesis; DNA pol I and DNA pol II are primarily required for repair.</p>
<p id="fs-id1676042">How does the replication machinery know where to begin? It turns out that there are specific nucleotide sequences called origins of replication where replication begins. In <em data-effect="italics">E. coli, </em>which has a single origin of replication on its one chromosome (as do most prokaryotes), it is approximately 245 base pairs long and is rich in AT sequences. The origin of replication is recognized by certain proteins that bind to this site. An enzyme called <span data-type="term">helicase</span> unwinds the DNA by breaking the hydrogen bonds between the nitrogenous base pairs. ATP hydrolysis is required for this process. As the DNA opens up, Y-shaped structures called <span data-type="term">replication forks</span> are formed. Two replication forks are formed at the origin of replication and these get extended bi- directionally as replication proceeds. <span data-type="term">Single-strand binding proteins</span> coat the single strands of DNA near the replication fork to prevent the single-stranded DNA from winding back into a double helix. DNA polymerase is able to add nucleotides only in the 5' to 3' direction (a new DNA strand can be only extended in this direction). It also requires a free 3'-OH group to which it can add nucleotides by forming a phosphodiester bond between the 3'-OH end and the 5' phosphate of the next nucleotide. This essentially means that it cannot add nucleotides if a free 3'-OH group is not available. Then how does it add the first nucleotide? The problem is solved with the help of a primer that provides the free 3'-OH end. Another enzyme, RNA <span data-type="term">primase</span>, synthesizes an RNA primer that is about five to ten nucleotides long and complementary to the DNA. Because this sequence primes the DNA synthesis, it is appropriately called the <span data-type="term">primer</span>. DNA polymerase can now extend this RNA primer, adding nucleotides one by one that are complementary to the template strand (<a href="#fig-ch14_04_01" class="autogenerated-content">[link]</a>).</p>
<div data-type="note" data-has-label="true" id="fs-id1305287" class="note art-connection" data-label=""><div data-type="title" class="title">Art Connection</div>
<p id="fs-id2979157">
<figure id="fig-ch14_04_01"><figcaption>A replication fork is formed when helicase separates the DNA strands at the origin of replication. The DNA tends to become more highly coiled ahead of the replication fork. Topoisomerase breaks and reforms DNA&#8217;s phosphate backbone ahead of the replication fork, thereby relieving the pressure that results from this supercoiling. Single-strand binding proteins bind to the single-stranded DNA to prevent the helix from re-forming. Primase synthesizes an RNA primer. DNA polymerase III uses this primer to synthesize the daughter DNA strand. On the leading strand, DNA is synthesized continuously, whereas on the lagging strand, DNA is synthesized in short stretches called Okazaki fragments. DNA polymerase I replaces the RNA primer with DNA. DNA ligase seals the gaps between the Okazaki fragments, joining the fragments into a single DNA molecule. (credit: modification of work by Mariana Ruiz Villareal)</figcaption><span data-type="media" id="fs-id2897265" data-alt="Illustration shows the replication fork. Helicase unwinds the helix, and single-strand binding proteins prevent the helix from re-forming. Topoisomerase prevents the DNA from getting too tightly coiled ahead of the replication fork. DNA primase forms an RNA primer, and DNA polymerase extends the DNA strand from the RNA primer. DNA synthesis occurs only in the 5' to 3' direction. On the leading strand, DNA synthesis occurs continuously. On the lagging strand, DNA synthesis restarts many times as the helix unwinds, resulting in many short fragments called &#8220;Okazaki fragments.&#8221; DNA ligase joins the Okazaki fragments together into a single DNA molecule.">
<img src="/resources/840f9f0df6390344a02b9719780d079af531d7c1/Figure_14_04_01.jpg" data-media-type="image/png" alt="Illustration shows the replication fork. Helicase unwinds the helix, and single-strand binding proteins prevent the helix from re-forming. Topoisomerase prevents the DNA from getting too tightly coiled ahead of the replication fork. DNA primase forms an RNA primer, and DNA polymerase extends the DNA strand from the RNA primer. DNA synthesis occurs only in the 5' to 3' direction. On the leading strand, DNA synthesis occurs continuously. On the lagging strand, DNA synthesis restarts many times as the helix unwinds, resulting in many short fragments called &#8220;Okazaki fragments.&#8221; DNA ligase joins the Okazaki fragments together into a single DNA molecule." width="400"/>
</span>
</figure></p>
<p id="fs-id1368727">You isolate a cell strain in which the joining together of Okazaki fragments is impaired and suspect that a mutation has occurred in an enzyme found at the replication fork. Which enzyme is most likely to be mutated?</p>
</div><p id="fs-id1385221">The replication fork moves at the rate of 1000 nucleotides per second. DNA polymerase can only extend in the 5' to 3' direction, which poses a slight problem at the replication fork. As we know, the DNA double helix is anti-parallel; that is, one strand is in the 5' to 3' direction and the other is oriented in the 3' to 5' direction. One strand, which is complementary to the 3' to 5' parental DNA strand, is synthesized continuously towards the replication fork because the polymerase can add nucleotides in this direction. This continuously synthesized strand is known as the <span data-type="term">leading strand</span>. The other strand, complementary to the 5' to 3' parental DNA, is extended away from the replication fork, in small fragments known as <span data-type="term">Okazaki fragments</span>, each requiring a primer to start the synthesis. Okazaki fragments are named after the Japanese scientist who first discovered them. The strand with the Okazaki fragments is known as the <span data-type="term">lagging strand</span>.</p>
<p id="fs-id2962574">The leading strand can be extended by one primer alone, whereas the lagging strand needs a new primer for each of the short Okazaki fragments. The overall direction of the lagging strand will be 3' to 5', and that of the leading strand 5' to 3'. A protein called the <span data-type="term">sliding clamp</span> holds the DNA polymerase in place as it continues to add nucleotides. The sliding clamp is a ring-shaped protein that binds to the DNA and holds the polymerase in place. <span data-type="term">Topoisomerase</span> prevents the over-winding of the DNA double helix ahead of the replication fork as the DNA is opening up; it does so by causing temporary nicks in the DNA helix and then resealing it. As synthesis proceeds, the RNA primers are replaced by DNA. The primers are removed by the exonuclease activity of DNA pol I, and the gaps are filled in by deoxyribonucleotides. The nicks that remain between the newly synthesized DNA (that replaced the RNA primer) and the previously synthesized DNA are sealed by the enzyme DNA <span data-type="term">ligase</span> that catalyzes the formation of phosphodiester linkage between the 3'-OH end of one nucleotide and the 5' phosphate end of the other fragment.</p>
<p id="fs-id1752302">Once the chromosome has been completely replicated, the two DNA copies move into two different cells during cell division. The process of DNA replication can be summarized as follows:</p>
    <ol id="fs-id2017788">    
<li>DNA unwinds at the origin of replication.</li>
        <li>Helicase opens up the DNA-forming replication forks; these are extended bidirectionally.</li>
        <li>Single-strand binding proteins coat the DNA around the replication fork to prevent rewinding of the DNA.</li>
        <li>Topoisomerase binds at the region ahead of the replication fork to prevent supercoiling.</li>
        <li>Primase synthesizes RNA primers complementary to the DNA strand.</li>
        <li>DNA polymerase starts adding nucleotides to the 3'-OH end of the primer.</li>
        <li>Elongation of both the lagging and the leading strand continues.</li>
        <li>RNA primers are removed by exonuclease activity.</li>
        <li>Gaps are filled by DNA pol by adding dNTPs.</li>
        <li>The gap between the two DNA fragments is sealed by DNA ligase, which helps in the formation of phosphodiester bonds.</li>
        </ol>
    <p id="fs-id2235232"><div data-type="newline"><br/></div><a href="#tab-ch14_04_01" class="autogenerated-content">[link]</a> summarizes the enzymes involved in prokaryotic DNA replication and the functions of each.</p><table id="tab-ch14_04_01" class="  " summary=""><thead>
                <tr><th colspan="2" data-align="left">Prokaryotic DNA Replication: Enzymes and Their Function</th></tr>
<tr><th>Enzyme/protein</th><th>Specific Function</th></tr>
            </thead><tbody>            
<tr><td>DNA pol I</td><td>Exonuclease activity removes RNA primer and replaces with newly synthesized DNA</td></tr>
    <tr><td>DNA pol II</td><td>Repair function</td></tr>
    <tr><td>DNA pol III</td><td>Main enzyme that adds nucleotides in the 5'-3' direction</td></tr>
    <tr><td>Helicase</td><td>Opens the DNA helix by breaking hydrogen bonds between the nitrogenous bases</td></tr>
    <tr><td>Ligase</td><td>Seals the gaps between the Okazaki fragments to create one continuous DNA strand</td></tr>
    <tr><td>Primase</td><td>Synthesizes RNA primers needed to start replication</td></tr>
    <tr><td>Sliding Clamp</td><td>Helps to hold the DNA polymerase in place when nucleotides are being added</td></tr>
    <tr><td>Topoisomerase</td><td>Helps relieve the stress on DNA when unwinding by causing breaks and then resealing the DNA</td></tr>
<tr><td>Single-strand binding proteins (SSB)</td><td>Binds to single-stranded DNA to avoid DNA rewinding back.</td></tr>
</tbody></table>
    
<div data-type="note" data-has-label="true" id="fs-id1672176" class="note interactive" data-label=""><div data-type="title" class="title">Link to Learning</div>
<span data-type="media" id="fs-id2914259" data-alt="QR Code representing a URL">
    <img src="/resources/504c5685908e0d41f8b1e09504d35e7fce47b3e1/replication_DNA.png" data-media-type="image/png" alt="QR Code representing a URL" width="120"/>
</span>
    <p id="fs-id2010960">Review the full process of DNA replication <a href="http://openstaxcollege.org/l/replication_DNA" target="_window">here</a>.</p>
</div><section data-depth="1" id="fs-id1974676" class="summary"><h1 data-type="title">Section Summary</h1><p id="fs-id1437221">Replication in prokaryotes starts from a sequence found on the chromosome called the origin of replication&#8212;the point at which the DNA opens up. Helicase opens up the DNA double helix, resulting in the formation of the replication fork. Single-strand binding proteins bind to the single-stranded DNA near the replication fork to keep the fork open. Primase synthesizes an RNA primer to initiate synthesis by DNA polymerase, which can add nucleotides only in the 5' to 3' direction. One strand is synthesized continuously in the direction of the replication fork; this is called the leading strand. The other strand is synthesized in a direction away from the replication fork, in short stretches of DNA known as Okazaki fragments. This strand is known as the lagging strand. Once replication is completed, the RNA primers are replaced by DNA nucleotides and the DNA is sealed with DNA ligase, which creates phosphodiester bonds between the 3'-OH of one end and the 5' phosphate of the other strand.</p>
</section>
<section data-depth="1" id="fs-idm22300288" class="art-exercise"><h1 data-type="title">Art Connections</h1>
<div data-type="exercise" class="exercise" id="fs-idm104376352"><div data-type="problem" class="problem" id="fs-idm62690592">
<p id="fs-idp29378960"><a href="#fig-ch14_04_01" class="autogenerated-content">[link]</a> You isolate a cell strain in which the joining together of Okazaki fragments is impaired and suspect that a mutation has occurred in an enzyme found at the replication fork. Which enzyme is most likely to be mutated?</p>
</div>
<div data-type="solution" class="solution" id="fs-idp76857456" data-label="">
<p id="fs-idp125877360"><a href="#fig-ch14_04_01" class="autogenerated-content">[link]</a> DNA ligase, as this enzyme joins together Okazaki fragments.</p>
</div>
</div></section>
<section data-depth="1" id="fs-id1250810" class="multiple-choice"><h1 data-type="title">Review Questions</h1><div data-type="exercise" class="exercise" id="fs-id1341900">
<div data-type="problem" class="problem" id="fs-id1519179"><p id="fs-id2329090">Which of the following components is not involved during the formation of the replication fork?</p>
<ol id="fs-id1602896" data-number-style="lower-alpha">
<li>single-strand binding proteins</li>
<li>helicase</li>
<li>origin of replication</li>
<li>ligase</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id2345394" data-label="">
<p id="fs-id2643307">D</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id2075108">
<div data-type="problem" class="problem" id="fs-id1396394"><p id="fs-id2138308">Which of the following does the enzyme primase synthesize?</p>
<ol id="fs-id2228365" data-number-style="lower-alpha">
<li>DNA primer</li>
<li>RNA primer</li>
<li>Okazaki fragments</li>
<li>phosphodiester linkage</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id1910465" data-label="">
<p id="fs-id2010183">B</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1406784">
<div data-type="problem" class="problem" id="fs-id2019618"><p id="fs-id1798381">In which direction does DNA replication take place?</p>
<ol id="fs-id3241387" data-number-style="lower-alpha">
<li>5'-3'</li>
<li>3'-5'</li>
<li>5'</li>
<li>3'</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id1512007" data-label="">
<p id="fs-id2979133">A</p>
</div>
</div>
</section>
<section data-depth="1" id="fs-id1426493" class="free-response"><h1 data-type="title">Free Response</h1>
<div data-type="exercise" class="exercise" id="fs-id1286460">
<div data-type="problem" class="problem" id="fs-id1441996"><p id="fs-id2198448">DNA replication is bidirectional and discontinuous; explain your understanding of those concepts.</p></div>
<div data-type="solution" class="solution" id="fs-id2936597" data-label="">
<p id="fs-id1268735">At an origin of replication, two replication forks are formed that are extended in two directions. On the lagging strand, Okazaki fragments are formed in a discontinuous manner.</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1805432">
<div data-type="problem" class="problem" id="fs-id1569099"><p id="fs-id2257887">What are Okazaki fragments and how they are formed?</p></div>
<div data-type="solution" class="solution" id="fs-id2114708" data-label="">
<p id="fs-id2336863">Short DNA fragments are formed on the lagging strand synthesized in a direction away from the replication fork. These are synthesized by DNA pol.</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1720501">
<div data-type="problem" class="problem" id="fs-id2228028"><p id="fs-id1778374">If the rate of replication in a particular prokaryote is 900 nucleotides per second, how long would it take 1.2 million base pair genomes to make two copies?</p></div>
<div data-type="solution" class="solution" id="fs-id3165297" data-label="">
<p id="fs-id2279649">1333 seconds or 22.2 minutes.</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id2126300">
<div data-type="problem" class="problem" id="fs-id671101"><p id="fs-id1986681">Explain the events taking place at the replication fork. If the gene for helicase is mutated, what part of replication will be affected?</p></div>
<div data-type="solution" class="solution" id="fs-id1286063" data-label="">
<p id="fs-id1436147">At the replication fork, the events taking place are helicase action, binding of single-strand binding proteins, primer synthesis, and synthesis of new strands. If there is a mutated helicase gene, the replication fork will not be extended.</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1403064">
<div data-type="problem" class="problem" id="fs-id2115858"><p id="fs-id1387702">What is the role of a primer in DNA replication? What would happen if you forgot to add a primer in a tube containing the reaction mix for a DNA sequencing reaction?</p></div>
<div data-type="solution" class="solution" id="fs-id1986827" data-label="">
<p id="fs-id2385726">Primer provides a 3'-OH group for DNA pol to start adding nucleotides. There would be no reaction in the tube without a primer, and no bands would be visible on the electrophoresis.</p>
</div>
</div>
</section>
<div data-type="glossary"><h2 data-type="glossary-title">Glossary</h2>
<dl class="definition" id="fs-id1635058"><dt>helicase</dt> <dd id="fs-id1407439">during replication, this enzyme helps to open up the DNA helix by breaking the hydrogen bonds</dd></dl>
<dl class="definition" id="fs-id1479286"><dt>lagging strand</dt> <dd id="fs-id1715240">during replication, the strand that is replicated in short fragments and away from the replication fork</dd></dl>
<dl class="definition" id="fs-id1442893"><dt>leading strand</dt> <dd id="fs-id1506798">strand that is synthesized continuously in the 5'-3' direction which is synthesized in the direction of the replication fork</dd></dl>
<dl class="definition" id="fs-id2596155"><dt>ligase</dt> <dd id="fs-id1780732">enzyme that catalyzes the formation of a phosphodiester linkage between the 3' OH and 5' phosphate ends of the DNA</dd></dl>
<dl class="definition" id="fs-id3019610"><dt>Okazaki fragment</dt> <dd id="fs-id2073222">DNA fragment that is synthesized in short stretches on the lagging strand</dd></dl>
<dl class="definition" id="fs-id1594620"><dt>primase</dt> <dd id="fs-id1693666">enzyme that synthesizes the RNA primer; the primer is needed for DNA pol to start synthesis of a new DNA strand</dd></dl>
<dl class="definition" id="fs-id2270395"><dt>primer</dt> <dd id="fs-id1760012">short stretch of nucleotides that is required to initiate replication; in the case of replication, the primer has RNA nucleotides</dd></dl>
<dl class="definition" id="fs-id1444125"><dt>replication fork</dt> <dd id="fs-id2636461">Y-shaped structure formed during initiation of replication</dd></dl>
<dl class="definition" id="fs-id1715236"><dt>single-strand binding protein</dt> <dd id="fs-id2072586">during replication, protein that binds to the single-stranded DNA; this helps in keeping the two strands of DNA apart so that they may serve as templates</dd></dl>
<dl class="definition" id="fs-id1703233"><dt>sliding clamp</dt> <dd id="fs-id2590659">ring-shaped protein that holds the DNA pol on the DNA strand</dd></dl>
<dl class="definition" id="fs-id1958554"><dt>topoisomerase</dt> <dd id="fs-id1466731">enzyme that causes underwinding or overwinding of DNA when DNA replication is taking place</dd></dl>
</div></body>

</html>