<html xmlns="http://www.w3.org/1999/xhtml">
<head xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml"><title>Chromosomal Theory and Genetic Linkage</title><meta name="created-time" content="2012/08/23 12:24:38 +0000"/><meta name="revised-time" content="2015/01/05 12:37:34.096 US/Central"/><meta name="author" content="OpenStaxCollege"/><meta name="licensor" content="OSCRiceUniversity"/><meta name="license" content="http://creativecommons.org/licenses/by/4.0/"/><meta name="keywords" content="centiMorgan, chromosomal theory, chromosomal theory of inheritance, chromosome map, crossing over, crossover, gene distance, genetic linkage, genetic map, genetics, homologous recombination, inheritance, linked genes, meiosis, nonparental type, parental type, recombinant type, recombination, recombination frequency, Sutton's theory of inheritance, Thomas Hunt Morgan, three-point test cross, Walter Sutton, x-linked genes"/><meta name="subject" content="Science and Technology"/></head>

<body xmlns="http://www.w3.org/1999/xhtml" xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml" xmlns:qml="http://cnx.rice.edu/qml/1.0" xmlns:mod="http://cnx.rice.edu/#moduleIds" xmlns:bib="http://bibtexml.sf.net/" xmlns:data="http://dev.w3.org/html5/spec/#custom"><div data-type="document-title">Chromosomal Theory and Genetic Linkage</div><div data-type="abstract">By the end of this section, you will be able to:
<ul>
<li>Discuss Sutton&#8217;s Chromosomal Theory of Inheritance</li> 
<li>Describe genetic linkage</li>
<li>Explain the process of homologous recombination, or crossing over</li>
<li>Describe how chromosome maps are created</li>
<li>Calculate the distances between three genes on a chromosome using a three-point test cross</li>
</ul></div>
        <p id="fs-id2021443">Long before chromosomes were visualized under a microscope, the father of modern genetics, Gregor Mendel, began studying heredity in 1843. With the improvement of microscopic techniques during the late 1800s, cell biologists could stain and visualize subcellular structures with dyes and observe their actions during cell division and meiosis. With each mitotic division, chromosomes replicated, condensed from an amorphous (no constant shape) nuclear mass into distinct X-shaped bodies (pairs of identical sister chromatids), and migrated to separate cellular poles.</p>
        <section data-depth="1" id="fs-id2904526"><h1 data-type="title">Chromosomal Theory of Inheritance</h1>
            <p id="fs-id1406190">The speculation that chromosomes might be the key to understanding heredity led several scientists to examine Mendel&#8217;s publications and re-evaluate his model in terms of the behavior of chromosomes during mitosis and meiosis. In 1902, Theodor Boveri observed that proper embryonic development of sea urchins does not occur unless chromosomes are present. That same year, Walter Sutton observed the separation of chromosomes into daughter cells during meiosis (<a href="#fig-ch13_01_01" class="autogenerated-content">[link]</a>). Together, these observations led to the development of the <span data-type="term">Chromosomal Theory of Inheritance</span>, which identified chromosomes as the genetic material responsible for Mendelian inheritance.</p>
            <figure id="fig-ch13_01_01"><figcaption>(a) Walter Sutton and (b) Theodor Boveri are credited with developing the Chromosomal Theory of Inheritance, which states that chromosomes carry the unit of heredity (genes).</figcaption><span data-type="media" id="fs-id2011864" data-alt="Part a is a photo of Walter Sutton. Part b is a photo of Theodor Boveri.">
                <img src="/resources/90a3ba21d45abff161746eaad69ad190d43f3b29/Figure_13_01_01.jpg" data-media-type="image/jpg" alt="Part a is a photo of Walter Sutton. Part b is a photo of Theodor Boveri." width="275"/>
            </span>
                </figure><p id="fs-id1236139">The Chromosomal Theory of Inheritance was consistent with Mendel&#8217;s laws and was supported by the following observations:</p>
            <ul id="fs-id1770608">
                <li>During meiosis, homologous chromosome pairs migrate as discrete structures that are independent of other chromosome pairs.</li>
                <li>The sorting of chromosomes from each homologous pair into pre-gametes appears to be random.</li>
                <li>Each parent synthesizes gametes that contain only half of their chromosomal complement.</li>
                <li>Even though male and female gametes (sperm and egg) differ in size and morphology, they have the same number of chromosomes, suggesting equal genetic contributions from each parent.</li>
                <li>The gametic chromosomes combine during fertilization to produce offspring with the same chromosome number as their parents.</li>
            </ul>
            <p id="fs-id2335848">Despite compelling correlations between the behavior of chromosomes during meiosis and Mendel&#8217;s abstract laws, the Chromosomal Theory of Inheritance was proposed long before there was any direct evidence that traits were carried on chromosomes. Critics pointed out that individuals had far more independently segregating traits than they had chromosomes. It was only after several years of carrying out crosses with the fruit fly, <em data-effect="italics">Drosophila melanogaster</em>, that Thomas Hunt Morgan provided experimental evidence to support the Chromosomal Theory of Inheritance.</p>
        </section>
        <section data-depth="1" id="fs-id1396027"><h1 data-type="title">Genetic Linkage and Distances</h1>
            <p id="fs-id1780491">Mendel&#8217;s work suggested that traits are inherited independently of each other. Morgan identified a 1:1 correspondence between a segregating trait and the X chromosome, suggesting that the random segregation of chromosomes was the physical basis of Mendel&#8217;s model. This also demonstrated that linked genes disrupt Mendel&#8217;s predicted outcomes. The fact that each chromosome can carry many linked genes explains how individuals can have many more traits than they have chromosomes. However, observations by researchers in Morgan&#8217;s laboratory suggested that alleles positioned on the same chromosome were not always inherited together. During meiosis, linked genes somehow became unlinked.</p>
            <section data-depth="2" id="fs-id2571351"><h2 data-type="title">Homologous Recombination</h2>
                <p id="fs-id2047023">In 1909, Frans Janssen observed chiasmata&#8212;the point at which chromatids are in contact with each other and may exchange segments&#8212;prior to the first division of meiosis. He suggested that alleles become unlinked and chromosomes physically exchange segments. As chromosomes condensed and paired with their homologs, they appeared to interact at distinct points. Janssen suggested that these points corresponded to regions in which chromosome segments were exchanged. It is now known that the pairing and interaction between homologous chromosomes, known as synapsis, does more than simply organize the homologs for migration to separate daughter cells. When synapsed, homologous chromosomes undergo reciprocal physical exchanges at their arms in a process called <span data-type="term">homologous recombination</span>, or more simply, &#8220;crossing over.&#8221;</p>
                <p id="fs-id1468864">To better understand the type of experimental results that researchers were obtaining at this time, consider a heterozygous individual that inherited dominant maternal alleles for two genes on the same chromosome (such as<em data-effect="italics"> AB</em>) and two recessive paternal alleles for those same genes (such as <em data-effect="italics">ab</em>). If the genes are linked, one would expect this individual to produce gametes that are either <em data-effect="italics">AB</em> or <em data-effect="italics">ab</em> with a 1:1 ratio. If the genes are unlinked, the individual should produce <em data-effect="italics">AB</em>, <em data-effect="italics">Ab</em>, <em data-effect="italics">aB</em>, and <em data-effect="italics">ab</em> gametes with equal frequencies, according to the Mendelian concept of independent assortment. Because they correspond to new allele combinations, the genotypes Ab and aB are <span data-type="term">nonparental types</span> that result from homologous recombination during meiosis. <span data-type="term">Parental types</span> are progeny that exhibit the same allelic combination as their parents. Morgan and his colleagues, however, found that when such heterozygous individuals were test crossed to a homozygous recessive parent (<em data-effect="italics">AaBb</em> &#215; <em data-effect="italics">aabb</em>), both parental and nonparental cases occurred. For example, 950 offspring might be recovered that were either <em data-effect="italics">AaBb</em> or <em data-effect="italics">aabb</em>, but 50 offspring would also be obtained that were either <em data-effect="italics">Aabb</em> or <em data-effect="italics">aaBb</em>. These results suggested that linkage occurred most often, but a significant minority of offspring were the products of recombination.</p>
                <div data-type="note" data-has-label="true" id="fs-id1595360" class="note art-connection" data-label=""><div data-type="title" class="title">Art Connection</div>
                    <p id="fs-id1612804"><figure id="fig-ch13_01_02" class=" "><figcaption>Inheritance patterns of unlinked and linked genes are shown. In (a), two genes are located on different chromosomes so independent assortment occurs during meiosis. The offspring have an equal chance of being the parental type (inheriting the same combination of traits as the parents) or a nonparental type (inheriting a different combination of traits than the parents). In (b), two genes are very close together on the same chromosome so that no crossing over occurs between them. The genes are therefore always inherited together and all of the offspring are the parental type. In (c), two genes are far apart on the chromosome such that crossing over occurs during every meiotic event. The recombination frequency will be the same as if the genes were on separate chromosomes. (d) The actual recombination frequency of fruit fly wing length and body color that Thomas Morgan observed in 1912 was 17 percent. A crossover frequency between 0 percent and 50 percent indicates that the genes are on the same chromosome and crossover occurs some of the time.</figcaption><span data-type="media" id="fs-id1262550" data-alt="The illustration shows the possible inheritance patterns of linked and unlinked genes. The example used includes fruit fly body color and wing length. Fruit flies may have a dominant gray color (G) or a recessive black color (g). They may have dominant long wings (L) or recessive short wings (l). Three hypothetical inheritance patterns for a test cross between a heterozygous and a recessive fruit fly are shown, based on gene placement. The actual experimental results published by Thomas Hunt Morgan in 1912 are also shown. In the first hypothetical inheritance pattern in part a, the genes for the two characteristics are on different chromosomes. Independent assortment occurs so that the ratio of genotypes in the offspring  is 1 GgLl:1 ggll:1 Ggll:1 ggLl, and 50% of the offspring are nonparental types. In the second hypothetical inheritance pattern in part b, the genes are close together on the same chromosome so that no crossover occurs between them.  The ratio of genotypes is 1 GgLl:1 ggll, and none of the offspring are recombinant. In the third hypothetical inheritance pattern in part c, the genes are far apart on the same chromosome so that crossing over occurs 100% of the time. The ratio of genotypes is the same as for genes on two different chromosomes, and 50% of the offspring are recombinant, nonparental types. Part d shows that the number of offspring that Thomas Hunt Morgan actually observed was 965: 944: 206:185 (GgLl:ggll:Ggll:ggLl). Seventeen percent of the offspring were recombinant, indicating that the genes are on the same chromosome and crossing over occurs between them some of the time.">
                        <img src="/resources/3aa785752e831a6effedbe8715f5a88b4b85994d/Figure_13_01_02.jpg" data-media-type="image/jpg" alt="The illustration shows the possible inheritance patterns of linked and unlinked genes. The example used includes fruit fly body color and wing length. Fruit flies may have a dominant gray color (G) or a recessive black color (g). They may have dominant long wings (L) or recessive short wings (l). Three hypothetical inheritance patterns for a test cross between a heterozygous and a recessive fruit fly are shown, based on gene placement. The actual experimental results published by Thomas Hunt Morgan in 1912 are also shown. In the first hypothetical inheritance pattern in part a, the genes for the two characteristics are on different chromosomes. Independent assortment occurs so that the ratio of genotypes in the offspring  is 1 GgLl:1 ggll:1 Ggll:1 ggLl, and 50% of the offspring are nonparental types. In the second hypothetical inheritance pattern in part b, the genes are close together on the same chromosome so that no crossover occurs between them.  The ratio of genotypes is 1 GgLl:1 ggll, and none of the offspring are recombinant. In the third hypothetical inheritance pattern in part c, the genes are far apart on the same chromosome so that crossing over occurs 100% of the time. The ratio of genotypes is the same as for genes on two different chromosomes, and 50% of the offspring are recombinant, nonparental types. Part d shows that the number of offspring that Thomas Hunt Morgan actually observed was 965: 944: 206:185 (GgLl:ggll:Ggll:ggLl). Seventeen percent of the offspring were recombinant, indicating that the genes are on the same chromosome and crossing over occurs between them some of the time." width="370"/>
                    </span>
                        </figure></p>
                    <p id="fs-id2573656">In a test cross for two characteristics such as the one shown here, can the predicted frequency of recombinant offspring be 60 percent? Why or why not?</p><!-- <solution>
                        <para><link window="new"  target-id="fig-ch13_01_02" document=""/> No. The predicted frequency of recombinant offspring ranges from 0% (for linked traits) to 50% (for unlinked traits).</para>
                        </solution> -->
                </div></section>
            <section data-depth="2" id="fs-id1627133"><h2 data-type="title">Genetic Maps</h2>
                <p id="fs-id1645084">Janssen did not have the technology to demonstrate crossing over so it remained an abstract idea that was not widely accepted. Scientists thought chiasmata were a variation on synapsis and could not understand how chromosomes could break and rejoin. Yet, the data were clear that linkage did not always occur. Ultimately, it took a young undergraduate student and an &#8220;all-nighter&#8221; to mathematically elucidate the problem of linkage and recombination.</p>
                <p id="fs-id1684950">In 1913, Alfred Sturtevant, a student in Morgan&#8217;s laboratory, gathered results from researchers in the laboratory, and took them home one night to mull them over. By the next morning, he had created the first &#8220;chromosome map,&#8221; a linear representation of gene order and relative distance on a chromosome (<a href="#fig-ch13_01_03" class="autogenerated-content">[link]</a>).</p>
                <div data-type="note" data-has-label="true" id="fs-id1941826" class="note art-connection" data-label=""><div data-type="title" class="title">Art Connection</div>
                    <p id="fs-id2186799"><figure id="fig-ch13_01_03"><figcaption>This genetic map orders <em data-effect="italics">Drosophila</em> genes on the basis of recombination frequency.</figcaption><span data-type="media" id="fs-id1464025" data-alt="The illustration shows a Drosophila genetic map. The gene for aristae length occurs at 0 centimorgans, or cM. The gene for body color occurs at 48.5 cM. The gene for red versus cinnabar eye color occurs at 57.5 cM. The gene for wing length occurs at 65.5 cM, and the gene for red versus brown eye color occurs at 104.5 cM. One cM is equivalent to a recombination frequency of 0.01.">
                        <img src="/resources/d315fc103131f8d08f1cbfeec58afe47756fedd2/Figure_13_01_03.png" data-media-type="image/png" alt="The illustration shows a Drosophila genetic map. The gene for aristae length occurs at 0 centimorgans, or cM. The gene for body color occurs at 48.5 cM. The gene for red versus cinnabar eye color occurs at 57.5 cM. The gene for wing length occurs at 65.5 cM, and the gene for red versus brown eye color occurs at 104.5 cM. One cM is equivalent to a recombination frequency of 0.01." width="200"/>
                    </span>
                        </figure></p>
                    <p id="fs-id1785828">Which of the following statements is true?</p>
                    <ol id="fs-id1511608" data-number-style="lower-alpha"><li>Recombination of the body color and red/cinnabar eye alleles will occur more frequently than recombination of the alleles for wing length and aristae length.</li>
                        <li>Recombination of the body color and aristae length alleles will occur more frequently than recombination of red/brown eye alleles and the aristae length alleles.</li>
                        <li>Recombination of the gray/black body color and long/short aristae alleles will not occur.</li>
                        <li>Recombination of the red/brown eye and long/short aristae alleles will occur more frequently than recombination of the alleles for wing length and body color.</li>
                    </ol><!-- <solution>
                        <para><link window="new"  target-id="fig-ch13_01_03" document=""/>D</para>
                        </solution> -->
                </div><p id="fs-id2137494">As shown in <a href="#fig-ch13_01_03" class="autogenerated-content">[link]</a>, by using recombination frequency to predict genetic distance, the relative order of genes on chromosome 2 could be inferred. The values shown represent map distances in centimorgans (cM), which correspond to recombination frequencies (in percent). Therefore, the genes for body color and wing size were 65.5 &#8722; 48.5 = 17 cM apart, indicating that the maternal and paternal alleles for these genes recombine in 17 percent of offspring, on average.</p><p id="fs-id1951192">To construct a chromosome map, Sturtevant assumed that genes were ordered serially on threadlike chromosomes. He also assumed that the incidence of recombination between two homologous chromosomes could occur with equal likelihood anywhere along the length of the chromosome. Operating under these assumptions, Sturtevant postulated that alleles that were far apart on a chromosome were more likely to dissociate during meiosis simply because there was a larger region over which recombination could occur. Conversely, alleles that were close to each other on the chromosome were likely to be inherited together. The average number of crossovers between two alleles&#8212;that is, their <span data-type="term">recombination frequency</span>&#8212;correlated with their genetic distance from each other, relative to the locations of other genes on that chromosome. Considering the example cross between <em data-effect="italics">AaBb</em> and <em data-effect="italics">aabb</em> above, the frequency of recombination could be calculated as 50/1000 = 0.05. That is, the likelihood of a crossover between genes <em data-effect="italics">A/a</em> and <em data-effect="italics">B/b</em> was 0.05, or 5 percent. Such a result would indicate that the genes were definitively linked, but that they were far enough apart for crossovers to occasionally occur. Sturtevant divided his genetic map into map units, or <span data-type="term">centimorgans (cM)</span>, in which a recombination frequency of 0.01 corresponds to 1 cM.</p><p id="fs-id2100648">By representing alleles in a linear map, Sturtevant suggested that genes can range from being perfectly linked (recombination frequency = 0) to being perfectly unlinked (recombination frequency = 0.5) when genes are on different chromosomes or genes are separated very far apart on the same chromosome. Perfectly unlinked genes correspond to the frequencies predicted by Mendel to assort independently in a dihybrid cross. A recombination frequency of 0.5 indicates that 50 percent of offspring are recombinants and the other 50 percent are parental types. That is, every type of allele combination is represented with equal frequency. This representation allowed Sturtevant to additively calculate distances between several genes on the same chromosome. However, as the genetic distances approached 0.50, his predictions became less accurate because it was not clear whether the genes were very far apart on the same chromosome or on different chromosomes.</p><p id="fs-id1341900">In 1931, Barbara McClintock and Harriet Creighton demonstrated the crossover of homologous chromosomes in corn plants. Weeks later, homologous recombination in <em data-effect="italics">Drosophila </em>was demonstrated microscopically by Curt Stern. Stern observed several X-linked phenotypes that were associated with a structurally unusual and dissimilar X chromosome pair in which one X was missing a small terminal segment, and the other X was fused to a piece of the Y chromosome. By crossing flies, observing their offspring, and then visualizing the offspring&#8217;s chromosomes, Stern demonstrated that every time the offspring allele combination deviated from either of the parental combinations, there was a corresponding exchange of an X chromosome segment. Using mutant flies with structurally distinct X chromosomes was the key to observing the products of recombination because DNA sequencing and other molecular tools were not yet available. It is now known that homologous chromosomes regularly exchange segments in meiosis by reciprocally breaking and rejoining their DNA at precise locations.</p>
                <div data-type="note" data-has-label="true" id="fs-id2060998" class="note interactive" data-label=""><div data-type="title" class="title">Link to Learning</div>
                    <span data-type="media" id="fs-id2022792" data-alt="QR Code representing a URL">
                        <img src="/resources/cc3730bb38e4dce8c8a0be50d3838a4356372f4b/gene_crossover.png" data-media-type="image/png" alt="QR Code representing a URL" width="120"/>
                    </span>
                    <p id="fs-id2570945">Review Sturtevant&#8217;s process to create a genetic map on the basis of recombination frequencies <a href="http://openstaxcollege.org/l/gene_crossover" target="_window">here</a>.</p></div></section>
            <section data-depth="2" id="fs-id1961794"><h2 data-type="title">Mendel&#8217;s Mapped Traits</h2>
                <p id="fs-id1802180">Homologous recombination is a common genetic process, yet Mendel never observed it. Had he investigated both linked and unlinked genes, it would have been much more difficult for him to create a unified model of his data on the basis of probabilistic calculations. Researchers who have since mapped the seven traits investigated by Mendel onto the seven chromosomes of the pea plant genome have confirmed that all of the genes he examined are either on separate chromosomes or are sufficiently far apart as to be statistically unlinked. Some have suggested that Mendel was enormously lucky to select only unlinked genes, whereas others question whether Mendel discarded any data suggesting linkage. In any case, Mendel consistently observed independent assortment because he examined genes that were effectively unlinked.</p>
            </section>
        </section>
        <section data-depth="1" id="fs-id2336690" class="summary"><h1 data-type="title">Section Summary</h1><p id="fs-id1506798">The Chromosomal Theory of inheritance, proposed by Sutton and Boveri, states that chromosomes are the vehicles of genetic heredity. Neither Mendelian genetics nor gene linkage is perfectly accurate; instead, chromosome behavior involves segregation, independent assortment, and occasionally, linkage. Sturtevant devised a method to assess recombination frequency and infer the relative positions and distances of linked genes on a chromosome on the basis of the average number of crossovers in the intervening region between the genes. Sturtevant correctly presumed that genes are arranged in serial order on chromosomes and that recombination between homologs can occur anywhere on a chromosome with equal likelihood. Whereas linkage causes alleles on the same chromosome to be inherited together, homologous recombination biases alleles toward an inheritance pattern of independent assortment.</p>
        </section>
        <section data-depth="1" id="fs-idp60956688" class="art-exercise"><h1 data-type="title">Art Connections</h1>
            <div data-type="exercise" class="exercise" id="fs-idp49371568">
                <div data-type="problem" class="problem" id="fs-idp83566544">
                    <p id="fs-idp8315472"><a href="#fig-ch13_01_02" class="autogenerated-content">[link]</a> In a test cross for two characteristics such as the one shown here, can the predicted frequency of recombinant offspring be 60 percent? Why or why not?</p></div>
                <div data-type="solution" class="solution" id="fs-idp38697712" data-label="">
                    <p id="fs-idm9214224"><a href="#fig-ch13_01_02" class="autogenerated-content">[link]</a> No. The predicted frequency of recombinant offspring ranges from 0% (for linked traits) to 50% (for unlinked traits).</p>
                    </div>
            </div>
            <div data-type="exercise" class="exercise" id="fs-idp13107328">
                <div data-type="problem" class="problem" id="fs-idp196286160">
                    <p id="fs-idp48650576"><a href="#fig-ch13_01_03" class="autogenerated-content">[link]</a> Which of the following statements is true?</p>
                    <ol id="fs-idp149339680" data-number-style="lower-alpha"><li>Recombination of the body color and red/cinnabar eye alleles will occur more frequently than recombination of the alleles for wing length and aristae length.</li>
                        <li>Recombination of the body color and aristae length alleles will occur more frequently than recombination of red/brown eye alleles and the aristae length alleles.</li>
                        <li>Recombination of the gray/black body color and long/short aristae alleles will not occur.</li>
                        <li>Recombination of the red/brown eye and long/short aristae alleles will occur more frequently than recombination of the alleles for wing length and body color.</li>
                    </ol></div>
                    <div data-type="solution" class="solution" id="fs-idp119823280" data-label="">
                        <p id="fs-idm2755664"><a href="#fig-ch13_01_03" class="autogenerated-content">[link]</a> D</p>
                        </div>
            </div>
        </section>
<section data-depth="1" id="fs-id2321141" class="multiple-choice"><h1 data-type="title">Review Questions</h1><div data-type="exercise" class="exercise" id="fs-id1596619">
                <div data-type="problem" class="problem" id="fs-id1480619"><p id="fs-id1723705">X-linked recessive traits in humans (or in <em data-effect="italics">Drosophila</em>) are observed ________.</p>
                    <ol id="fs-id1894764" data-number-style="lower-alpha">
                        <li>in more males than females</li>
                        <li>in more females than males</li>
                        <li>in males and females equally</li>
                        <li>in different distributions depending on the trait</li>
                    </ol>
                </div>
                <div data-type="solution" class="solution" id="fs-id2570734" data-label="">
                    <p id="fs-id2075720">A</p>
                </div>
            </div>
            <div data-type="exercise" class="exercise" id="fs-id2165395">
                <div data-type="problem" class="problem" id="fs-id1803123"><p id="fs-id2190439">The first suggestion that chromosomes may physically exchange segments came from the microscopic identification of ________.</p>
                    <ol id="fs-id2574822" data-number-style="lower-alpha">
                        <li>synapsis</li>
                        <li>sister chromatids</li>
                        <li>chiasmata</li>
                        <li>alleles</li>
                    </ol>
                </div>
                <div data-type="solution" class="solution" id="fs-id1780811" data-label="">
                    <p id="fs-id1695753">C</p>
                </div>
            </div>
            <div data-type="exercise" class="exercise" id="fs-id2914799">
                <div data-type="problem" class="problem" id="fs-id1988877"><p id="fs-id2110663">Which recombination frequency corresponds to independent assortment and the absence of linkage?</p>
                    <ol id="fs-id2315538" data-number-style="lower-alpha">
                        <li>0</li>
                        <li>0.25</li>
                        <li>0.50</li>
                        <li>0.75</li>
                    </ol>
                </div>
                <div data-type="solution" class="solution" id="fs-id2155997" data-label="">
                    <p id="fs-id1236538">C</p>
                </div>
            </div>
            <div data-type="exercise" class="exercise" id="fs-id2310144">
                <div data-type="problem" class="problem" id="fs-id2897234"><p id="fs-id1720799">Which recombination frequency corresponds to perfect linkage and violates the law of independent assortment?</p>
                    <ol id="fs-id1798480" data-number-style="lower-alpha">
                        <li>0</li>
                        <li>0.25</li>
                        <li>0.50</li>
                        <li>0.75</li>
                    </ol>
                </div>
                <div data-type="solution" class="solution" id="fs-id2914634" data-label="">
                    <p id="fs-id1238553">A</p>
                </div>
            </div>
        </section>
        <section data-depth="1" id="fs-id1384518" class="free-response"><h1 data-type="title">Free Response</h1>
            <div data-type="exercise" class="exercise" id="fs-id2890873">
                <div data-type="problem" class="problem" id="fs-id2914426"><p id="fs-id2317257">Explain how the Chromosomal Theory of Inheritance helped to advance our understanding of genetics.</p></div>
                <div data-type="solution" class="solution" id="fs-id2491136" data-label="">
                    <p id="fs-id2321542">The Chromosomal Theory of Inheritance proposed that genes reside on chromosomes. The understanding that chromosomes are linear arrays of genes explained linkage, and crossing over explained recombination.</p>
                </div>
            </div>
        </section>
    <div data-type="glossary"><h2 data-type="glossary-title">Glossary</h2>
        <dl class="definition" id="fs-id2200120"><dt>centimorgan (cM)</dt> <dd id="fs-id1797882">(also, map unit) relative distance that corresponds to a recombination frequency of 0.01</dd></dl>
        <dl class="definition" id="fs-id1310227"><dt>Chromosomal Theory of Inheritance</dt> <dd id="fs-id1467930">theory proposing that chromosomes are the vehicles of genes and that their behavior during meiosis is the physical basis of the inheritance patterns that Mendel observed</dd></dl>
        <dl class="definition" id="fs-id2248874"><dt>homologous recombination</dt> <dd id="fs-id1957277">process by which homologous chromosomes undergo reciprocal physical exchanges at their arms, also known as crossing over</dd></dl>
        <dl class="definition" id="fs-id3241178"><dt>nonparental (recombinant) type</dt> <dd id="fs-id2013517">progeny resulting from homologous recombination that exhibits a different allele combination compared with its parents</dd></dl>
        <dl class="definition" id="fs-id2891394"><dt>parental types</dt> <dd id="fs-id1986668">progeny that exhibits the same allelic combination as its parents</dd></dl>
        <dl class="definition" id="fs-id2595875"><dt>recombination frequency</dt> <dd id="fs-id2026021">average number of crossovers between two alleles; observed as the number of nonparental types in a population of progeny</dd></dl>
    </div></body>

</html>