<html xmlns="http://www.w3.org/1999/xhtml">
<head xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml"><title>DNA Repair</title><meta name="created-time" content="2012/08/23 13:44:24 -0500"/><meta name="revised-time" content="2014/07/08 10:45:49.044 GMT-5"/><meta name="author" content="OpenStaxCollege"/><meta name="licensor" content="OSCRiceUniversity"/><meta name="license" content="http://creativecommons.org/licenses/by/4.0/"/><meta name="keywords" content="deoxyribonucleic acid, DNA, DNA function, DNA repair, DNA structure, induced mutation, mismatch repair, mutation, nucleotide excision repair, point mutation, proofreading, silent mutation, spontaneous mutation, transition substitution, transversion substitution"/><meta name="subject" content="Science and Technology"/></head>

<body xmlns="http://www.w3.org/1999/xhtml" xmlns:c="http://cnx.rice.edu/cnxml" xmlns:md="http://cnx.rice.edu/mdml" xmlns:qml="http://cnx.rice.edu/qml/1.0" xmlns:mod="http://cnx.rice.edu/#moduleIds" xmlns:bib="http://bibtexml.sf.net/" xmlns:data="http://dev.w3.org/html5/spec/#custom"><div data-type="document-title">DNA Repair</div><div data-type="abstract">By the end of this section, you will be able to:
<ul>
<li>Discuss the different types of mutations in DNA</li>
<li>Explain DNA repair mechanisms</li>
</ul></div>
<p id="fs-id1976493">DNA replication is a highly accurate process, but mistakes can occasionally occur, such as a DNA polymerase inserting a wrong base. Uncorrected mistakes may sometimes lead to serious consequences, such as cancer. Repair mechanisms correct the mistakes. In rare cases, mistakes are not corrected, leading to mutations; in other cases, repair enzymes are themselves mutated or defective.</p>
<p id="fs-id2348700">Most of the mistakes during DNA replication are promptly corrected by DNA polymerase by proofreading the base that has been just added (<a href="#fig-ch14_06_01" class="autogenerated-content">[link]</a>). In <span data-type="term">proofreading</span>, the DNA pol reads the newly added base before adding the next one, so a correction can be made. The polymerase checks whether the newly added base has paired correctly with the base in the template strand. If it is the right base, the next nucleotide is added. If an incorrect base has been added, the enzyme makes a cut at the phosphodiester bond and releases the wrong nucleotide. This is performed by the exonuclease action of DNA pol III. Once the incorrect nucleotide has been removed, a new one will be added again.</p>
<figure id="fig-ch14_06_01"><figcaption>Proofreading by DNA polymerase corrects errors during replication.</figcaption>
<span data-type="media" id="fs-id2321057" data-alt="Illustration shows DNA polymerase replicating a strand of DNA. The enzyme has accidentally inserted G opposite A, resulting in a bulge. The enzyme backs up to fix the error.">
<img src="/resources/5852dbd448ac763ad93c04a07024cb1c260e1d09/Figure_14_06_01.jpg" data-media-type="image/jpg" alt="Illustration shows DNA polymerase replicating a strand of DNA. The enzyme has accidentally inserted G opposite A, resulting in a bulge. The enzyme backs up to fix the error." width="320"/>
</span>
</figure>
<p id="fs-id2186525">Some errors are not corrected during replication, but are instead corrected after replication is completed; this type of repair is known as <span data-type="term">mismatch repair</span> (<a href="#fig-ch14_06_02" class="autogenerated-content">[link]</a>). The enzymes recognize the incorrectly added nucleotide and excise it; this is then replaced by the correct base. If this remains uncorrected, it may lead to more permanent damage. How do mismatch repair enzymes recognize which of the two bases is the incorrect one? In <em data-effect="italics">E. coli</em>, after replication, the nitrogenous base adenine acquires a methyl group; the parental DNA strand will have methyl groups, whereas the newly synthesized strand lacks them. Thus, DNA polymerase is able to remove the wrongly incorporated bases from the newly synthesized, non-methylated strand. In eukaryotes, the mechanism is not very well understood, but it is believed to involve recognition of unsealed nicks in the new strand, as well as a short-term continuing association of some of the replication proteins with the new daughter strand after replication has completed.</p>
<figure id="fig-ch14_06_02"><figcaption>In mismatch repair, the incorrectly added base is detected after replication. The mismatch repair proteins detect this base and remove it from the newly synthesized strand by nuclease action. The gap is now filled with the correctly paired base.</figcaption><span data-type="media" id="fs-id2025027" data-alt="The top illustration shows a replicated DNA strand with G-T base mismatch. The bottom illustration shows the repaired DNA, which has the correct G-C base pairing.">
<img src="/resources/b6d0aac3e8715b029491bf3c5d76dfaa19c8bf98/Figure_14_06_02.jpg" data-media-type="image/jpg" alt="The top illustration shows a replicated DNA strand with G-T base mismatch. The bottom illustration shows the repaired DNA, which has the correct G-C base pairing." width="250"/>
</span>
</figure><p id="fs-id3000290">In another type of repair mechanism, <span data-type="term">nucleotide excision repair</span>, enzymes replace incorrect bases by making a cut on both the 3' and 5' ends of the incorrect base (<a href="#fig-ch14_06_03" class="autogenerated-content">[link]</a>). The segment of DNA is removed and replaced with the correctly paired nucleotides by the action of DNA pol. Once the bases are filled in, the remaining gap is sealed with a phosphodiester linkage catalyzed by DNA ligase. This repair mechanism is often employed when UV exposure causes the formation of pyrimidine dimers.</p>
<figure id="fig-ch14_06_03"><figcaption>Nucleotide excision repairs thymine dimers. When exposed to UV, thymines lying adjacent to each other can form thymine dimers. In normal cells, they are excised and replaced.</figcaption><span data-type="media" id="fs-id1695253" data-alt="Illustration shows a DNA strand in which a thymine dimer has formed. Excision repair enzyme cut out the section of DNA that contains the dimer so it can be replaced with normal base pairs.">
<img src="/resources/1ed5e526bed52da984fdab246ef220ae23d04197/Figure_14_06_03.jpg" data-media-type="image/jpg" alt="Illustration shows a DNA strand in which a thymine dimer has formed. Excision repair enzyme cut out the section of DNA that contains the dimer so it can be replaced with normal base pairs." width="200"/>
</span>
</figure><p id="fs-id2078153">A well-studied example of mistakes not being corrected is seen in people suffering from xeroderma pigmentosa (<a href="#fig-ch14_06_04" class="autogenerated-content">[link]</a>). Affected individuals have skin that is highly sensitive to UV rays from the sun. When individuals are exposed to UV, pyrimidine dimers, especially those of thymine, are formed; people with xeroderma pigmentosa are not able to repair the damage. These are not repaired because of a defect in the nucleotide excision repair enzymes, whereas in normal individuals, the thymine dimers are excised and the defect is corrected. The thymine dimers distort the structure of the DNA double helix, and this may cause problems during DNA replication. People with xeroderma pigmentosa may have a higher risk of contracting skin cancer than those who dont have the condition.</p><figure id="fig-ch14_06_04"><figcaption>Xeroderma pigmentosa is a condition in which thymine dimerization from exposure to UV is not repaired.  Exposure to sunlight results in skin lesions. (credit: James Halpern et al.)</figcaption><span data-type="media" id="fs-id2583496" data-alt="Photo shows a person with mottled skin lesions that result from xermoderma pigmentosa.">
<img src="/resources/17e534b2bf272de15b8d1f9083c77d52c1d20e1f/Figure_14_06_04.jpg" data-media-type="image/jpg" alt="Photo shows a person with mottled skin lesions that result from xermoderma pigmentosa." width="320"/>
</span>
</figure><p id="fs-id1310247">Errors during DNA replication are not the only reason why mutations arise in DNA. <span data-type="term">Mutations</span>, variations in the nucleotide sequence of a genome, can also occur because of damage to DNA. Such mutations may be of two types: induced or spontaneous. <span data-type="term">Induced mutations</span> are those that result from an exposure to chemicals, UV rays, x-rays, or some other environmental agent. <span data-type="term">Spontaneous mutations</span> occur without any exposure to any environmental agent; they are a result of natural reactions taking place within the body.</p>
<p id="fs-id1368727">Mutations may have a wide range of effects. Some mutations are not expressed; these are known as <span data-type="term">silent mutations</span>. <span data-type="term">Point mutations</span> are those mutations that affect a single base pair. The most common nucleotide mutations are substitutions, in which one base is replaced by another. These can be of two types, either transitions or transversions. <span data-type="term">Transition substitution</span> refers to a purine or pyrimidine being replaced by a base of the same kind; for example, a purine such as adenine may be replaced by the purine guanine. <span data-type="term">Transversion substitution</span> refers to a purine being replaced by a pyrimidine, or vice versa; for example, cytosine, a pyrimidine, is replaced by adenine, a purine. Mutations can also be the result of the addition of a base, known as an insertion, or the removal of a base, also known as deletion. Sometimes a piece of DNA from one chromosome may get translocated to another chromosome or to another region of the same chromosome; this is also known as translocation. These mutation types are shown in <a href="#fig-ch14_06_05" class="autogenerated-content">[link]</a>.</p>
<div data-type="note" data-has-label="true" id="fs-id2046784" class="note art-connection" data-label=""><div data-type="title" class="title">Art Connection</div>
<p id="fs-id2216817"><figure id="fig-ch14_06_05"><figcaption>Mutations can lead to changes in the protein sequence encoded by the DNA.</figcaption><span data-type="media" id="fs-id2221137" data-alt="Illustration shows different types of point mutations that result from a single amino acid substitution. In a silent mutation, no change in the amino acid sequence occurs. In a missense mutation, one amino acid is substituted for another. In a nonsense mutation, a stop codon is substituted for an amino acid. In a frameshift mutation, one or more bases is added or deleted, resulting in a change in the reading frame.">
<img src="/resources/7e47a8dfec12b8cbda375c7071482974a87793d2/Figure_14_06_05.png" data-media-type="image/png" alt="Illustration shows different types of point mutations that result from a single amino acid substitution. In a silent mutation, no change in the amino acid sequence occurs. In a missense mutation, one amino acid is substituted for another. In a nonsense mutation, a stop codon is substituted for an amino acid. In a frameshift mutation, one or more bases is added or deleted, resulting in a change in the reading frame." width="325"/>
</span>
</figure></p>
<p id="fs-id1813148">A frameshift mutation that results in the insertion of three nucleotides is often less deleterious than a mutation that results in the insertion of one nucleotide. Why?</p>
</div><p id="fs-id2936137">Mutations in repair genes have been known to cause cancer. Many mutated repair genes have been implicated in certain forms of pancreatic cancer, colon cancer, and colorectal cancer. Mutations can affect either somatic cells or germ cells. If many mutations accumulate in a somatic cell, they may lead to problems such as the uncontrolled cell division observed in cancer. If a mutation takes place in germ cells, the mutation will be passed on to the next generation, as in the case of hemophilia and xeroderma pigmentosa.</p><section data-depth="1" id="fs-id2065835" class="summary"><h1 data-type="title">Section Summary</h1><p id="fs-id1242469">DNA polymerase can make mistakes while adding nucleotides. It edits the DNA by proofreading every newly added base. Incorrect bases are removed and replaced by the correct base, and then a new base is added. Most mistakes are corrected during replication, although when this does not happen, the mismatch repair mechanism is employed. Mismatch repair enzymes recognize the wrongly incorporated base and excise it from the DNA, replacing it with the correct base. In yet another type of repair, nucleotide excision repair, the incorrect base is removed along with a few bases on the 5' and 3' end, and these are replaced by copying the template with the help of DNA polymerase. The ends of the newly synthesized fragment are attached to the rest of the DNA using DNA ligase, which creates a phosphodiester bond.</p>
<p id="fs-id2595588">Most mistakes are corrected, and if they are not, they may result in a mutation defined as a permanent change in the DNA sequence. Mutations can be of many types, such as substitution, deletion, insertion, and translocation. Mutations in repair genes may lead to serious consequences such as cancer. Mutations can be induced or may occur spontaneously.</p>
</section>
<section data-depth="1" id="fs-idp123953472" class="art-exercise"><h1 data-type="title">Art Connections</h1>
    <div data-type="exercise" class="exercise" id="fs-idp163052304">
        <div data-type="problem" class="problem" id="fs-idp152472016">
            <p id="fs-idp131842896"><a href="#fig-ch14_06_05" class="autogenerated-content">[link]</a> A frameshift mutation that results in the insertion of three nucleotides is often less deleterious than a mutation that results in the insertion of one nucleotide. Why?</p>
            </div>
        <div data-type="solution" class="solution" id="fs-idp122564624">
            <p id="fs-idp74157744"><a href="#fig-ch14_06_05" class="autogenerated-content">[link]</a> If three nucleotides are added, one additional amino acid will be incorporated into the protein chain, but the reading frame wont shift.</p></div>
    </div>
</section>
<section data-depth="1" id="fs-id2681952" class="multiple-choice"><h1 data-type="title">Review Questions</h1><div data-type="exercise" class="exercise" id="fs-id1951731">
<div data-type="problem" class="problem" id="fs-id1477381"><p id="fs-id1685167">During proofreading, which of the following enzymes reads the DNA?</p>
<ol id="fs-id2781180" data-number-style="lower-alpha">
<li>primase</li>
<li>topoisomerase</li>
<li>DNA pol</li>
<li>helicase</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id2750351" data-label="">
<p id="fs-id1477929">C</p>
</div>
</div>
<div data-type="exercise" class="exercise" id="fs-id1414909">
<div data-type="problem" class="problem" id="fs-id2415145"><p id="fs-id2073382">The initial mechanism for repairing nucleotide errors in DNA is ________.</p>
<ol id="fs-id1794075" data-number-style="lower-alpha">
<li>mismatch repair</li>
<li>DNA polymerase proofreading</li>
<li>nucleotide excision repair</li>
<li>thymine dimers</li>
</ol>
</div>
<div data-type="solution" class="solution" id="fs-id1837280" data-label="">
<p id="fs-id2051151">B</p>
</div>
</div>
</section>
<section data-depth="1" id="fs-id811872" class="free-response"><h1 data-type="title">Free Response</h1>
<div data-type="exercise" class="exercise" id="fs-id2008893">
<div data-type="problem" class="problem" id="fs-id2763212"><p id="fs-id3044550">What is the consequence of mutation of a mismatch repair enzyme? How will this affect the function of a gene?</p></div>
<div data-type="solution" class="solution" id="fs-id1385923" data-label="">
<p id="fs-id1712102">Mutations are not repaired, as in the case of xeroderma pigmentosa. Gene function may be affected or it may not be expressed.</p>
</div>
</div>
</section>
<div data-type="glossary"><h2 data-type="glossary-title">Glossary</h2>
<dl class="definition" id="fs-id1719569"><dt>induced mutation</dt> <dd id="fs-id1769884">mutation that results from exposure to chemicals or environmental agents</dd></dl>
<dl class="definition" id="fs-id2199830"><dt>mutation</dt> <dd id="fs-id2905083">variation in the nucleotide sequence of a genome</dd></dl>
<dl class="definition" id="fs-id2019618"><dt>mismatch repair</dt> <dd id="fs-id2308429">type of repair mechanism in which mismatched bases are removed after replication</dd></dl>
<dl class="definition" id="fs-id2013508"><dt>nucleotide excision repair</dt> <dd id="fs-id2689408">type of DNA repair mechanism in which the wrong base, along with a few nucleotides upstream or downstream, are removed</dd></dl>
<dl class="definition" id="fs-id1770263"><dt>proofreading</dt> <dd id="fs-id1785888">function of DNA pol in which it reads the newly added base before adding the next one</dd></dl>
<dl class="definition" id="fs-id1560334"><dt>point mutation</dt> <dd id="fs-id2186102">mutation that affects a single base</dd></dl>
<dl class="definition" id="fs-id1582545"><dt>silent mutation</dt> <dd id="fs-id1595608">mutation that is not expressed</dd></dl>
<dl class="definition" id="fs-id1412270"><dt>spontaneous mutation</dt> <dd id="fs-id2926691">mutation that takes place in the cells as a result of chemical reactions taking place naturally without exposure to any external agent</dd></dl>
<dl class="definition" id="fs-id2137494"><dt>transition substitution</dt> <dd id="fs-id1693442">when a purine is replaced with a purine or a pyrimidine is replaced with another pyrimidine</dd></dl>
<dl class="definition" id="fs-id1450506"><dt>transversion substitution</dt> <dd id="fs-id1506870">when a purine is replaced by a pyrimidine or a pyrimidine is replaced by a purine</dd></dl>
</div></body>

</html>